crypto/src/main/kotlin/web5/sdk/crypto/Crypto.kt

```diff
@@ -106,7 +103,11 @@ public object Crypto {
     val rawCurve = privateKey.toJSONObject()["crv"]
     val curve = rawCurve?.let { Curve.parse(it.toString()) }
 
-    val signer = getSigner(privateKey.algorithm, curve)
+    val signer = if (curve == Curve.SECP256K1 && privateKey.algorithm == null) {
```

Comment:
dont understand this, maybe a comment on why if the alg is null we hardcode it

Reply:
Reverted this changes in favor of adding them in the mapping.